examples/trading/frontend/src/components/escrows/CreateEscrow.tsx

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import { ApiLockedObject } from "@/types/types";
+import { useCurrentAccount, useSuiClientInfiniteQuery } from "@mysten/dapp-kit";
+import { formatAddress } from "@mysten/sui.js/utils";
+import { Avatar, Button, Select } from "@radix-ui/themes";
+import { InfiniteScrollArea } from "@/components/InfiniteScrollArea";
+import { useState } from "react";
+import { ObjectLink } from "../ObjectLink";
+import { useCreateEscrowMutation } from "@/mutations/escrow";
+
+export function CreateEscrow({ locked }: { locked: ApiLockedObject }) {
+  const [objectId, setObjectId] = useState<string | undefined>(undefined);
+  const account = useCurrentAccount();
+
+  const { mutate: createEscrowMutation, isPending } = useCreateEscrowMutation();
+
+  const { data, fetchNextPage, isFetchingNextPage, hasNextPage, refetch } =
+    useSuiClientInfiniteQuery(
+      "getOwnedObjects",
+      {
+        owner: account?.address!,
+        options: {
+          showDisplay: true,
+          showType: true,
+        },
+      },
+      {
+        enabled: !!account,
+        select: (data) =>
+          data.pages
+            .flatMap((page) => page.data)
+            .filter(
+              (x) => !!x.data?.display && !!x.data?.display?.data?.image_url,
+            ),
+      },
+    );
+
+  const getObject = () => {
+    const object = data?.find((x) => x.data?.objectId === objectId);
+
+    if (!object || !object.data) {
+      return;
+    }
+    return object.data;
+  };
+
+  return (
+    <div className="px-3 py-3 grid grid-cols-1 gap-5  mt-3 rounded">
```

Comment:
extremely minor nit: double space 😅 

```suggestion
    <div className="px-3 py-3 grid grid-cols-1 gap-5 mt-3 rounded">
```

Reply:
Haha, love this attention to detail